src/Microsoft.OpenApi.Readers/ParseNodes/ListNode.cs

```diff
@@ -38,6 +38,20 @@ public override List<T> CreateList<T>(Func<MapNode, T> map)
                 .ToList();
         }
 
+        public override List<IOpenApiAny> CreateListOfAny()
+        {
+            var yamlSequence = _nodeList;
```

Comment:
yamlSequence is not necessary. Why can't use the _nodeList directly?

Reply:
Fixed.